setup.py

```diff
@@ -12,6 +12,7 @@
     "python-binary-memcached",
     "rio-color",
     "rio-tiler-crs==3.0b2",
+    "rio-cogeo",
```

Comment:
I assume rio-cogeo isn't a large dependency?

Reply:
no it's not